Dockerfile.elixir

```diff
@@ -0,0 +1,16 @@
+FROM rust:1.49.0 AS build
```

Comment:
There is no problem that the build step is present in both Dockerfile.elixir and Dockerfile.erlang.
Layer caching is smart enough to do the work only once.

Reply:
Nice! I might change this though to use the statically linked binary created by the release so there's no compilation.